idb/postgres/postgres_integration_test.go

```diff
@@ -2339,7 +2356,9 @@ func TestDeleteTransactions(t *testing.T) {
 	for row := range rowsCh {
 		require.NoError(t, row.Error)
 		require.NotNil(t, row.Txn)
-		assert.Equal(t, txns[i], *row.Txn)
+		expected := base64.StdEncoding.EncodeToString(msgpack.Encode(txns[i]))
+		actual := base64.StdEncoding.EncodeToString(msgpack.Encode(*row.Txn))
```

Comment:
Is this because the types are different? This is a nice confidence builder that the SDK is fully compatible with go-algorand

Reply:
yes. I expected these to change again with the processor updates.